src/bin/coldsnap/main.rs

```diff
@@ -35,9 +32,52 @@ async fn main() {
 
 async fn run() -> Result<()> {
     let args: Args = argh::from_env();
+    let config: aws_config::ConfigLoader = match (args.region, &args.profile) {
+        (Some(region), _) => {
+            // Region option passed in
+            aws_config::from_env().region(Region::new(region))
```

Comment:
I use an EC2 instance for development that has a fairly limited instance role assigned, and an `~/.aws/config` file with a more powerful profile for use as needed.

Just want to confirm that this scenario still works.

Reply:
I called this without having an aws config on an EC2 instance with an instance profile, and also tested the profiles after I created them on the same EC2 instance.